README.md

```diff
@@ -435,6 +435,30 @@ To create a different backup and use you can do:
 # restic_backup.sh
 ```
 
+### Optional: Desktop Notifications
```

Comment:
Hmm confusing. THis exist in master branch, so why does it show up as an addition in this PR? 🤔

Reply:
Something weird happened when I rebased, I suspect it is because I did the rebase on my fork, which was not fast enough to sync with erikw/master. Then something weird happened. I'm not sure, but i'll fix it